src/script/interpreter.cpp

```diff
@@ -1536,11 +1538,25 @@ void PrecomputedTransactionData::Init(const T& txTo, std::vector<CTxOut>&& spent
         if (uses_bip341_taproot && uses_bip143_segwit) break; // No need to scan further if we already need all.
     }
 
-    if (uses_bip143_segwit || uses_bip341_taproot) {
+    if (uses_bip143_segwit || uses_bip341_taproot || uses_bip119_ctv) {
         // Computations shared between both sighash schemes.
```

Comment:
maybe just nix this comment entirely?

Reply:
Yeah, changing it to "all" doesn't seem correct either.

As long as `uses_bip119_ctv=true` unconditionally, it's the same behavior - but I think it would be more in the spirit of the existing code to move the calculation of `m_scriptSigs_single_hash` and `m_standard_template_single_hash` out of this conditional to a conditional only dependent on `uses_bip119_ctv`, as is done for segwit-specific or taproot-specific hashes.

Reply:
the approach for this has been changed in follow up patches, so the comment can stay as is :)